src/components/ui/TaskFooter.jsx

```diff
@@ -32,12 +38,14 @@ export default function TaskFooter({ isCompleted = false, taskId }) {
           <DeleteIcon />
         </Button>
       </div>
-      {isCompleted && <div className="task__footer__right">completed in</div>}
+      {completedAt && (
+        <div className="task__footer__right">completed in: {daysBetweenDate(completedAt)}days</div>
+      )}
     </div>
   );
 }
 
 TaskFooter.propTypes = {
-  isCompleted: propTypes.bool,
+  completedAt: propTypes.string,
```

Comment:
is it date type? if it is can we validate it is instance of a date class?

Reply:
Bhai date type used for prop validation